src/uwtools/api/driver.py

```diff
@@ -0,0 +1,28 @@
+"""
+API access to the ``uwtools`` driver base classes.
+"""
+
+import importlib
+import sys
+
+_CLASSNAMES = [
+    "Assets",
+    "AssetsCycleBased",
+    "AssetsCycleAndLeadtimeBased",
+    "AssetsTimeInvariant",
+    "Driver",
+    "DriverCycleBased",
+    "DriverCycleAndLeadtimeBased",
+    "DriverTimeInvariant",
+]
+
+
+def _add_classes():
+    m = importlib.import_module("uwtools.drivers.driver")
+    for classname in _CLASSNAMES:
+        setattr(sys.modules[__name__], classname, getattr(m, classname))
+        __all__.append(classname)
+
+
+__all__: list[str] = []
+_add_classes()
```

Comment:
This module defines the driver base classes to expose to the API and adds them to this module.